sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -22,6 +22,15 @@ import { TokenType } from '@azure/core-amqp';
 import { WebSocketImpl } from 'rhea-promise';
 import { WebSocketOptions } from '@azure/core-amqp';
 
+// @public
+export type AllowUndefinedFieldsForQueue = "forwardTo" | "authorizationRules" | "forwardDeadLetteredMessagesTo";
```

Comment:
Am guessing this is because the service can return `undefined` for these?
If so, then the name of the type can be misleading here. At a quick glance on just the type, I took this to also mean that these are the only fields which can have undefined when creating/updating the queue

Reply:
Ah, I couldn't think of a better name, whatever I do will make it even longer, any suggestions?

Reply:
Can be ignored since it is removed completely.